stdlib/LLVMLibUnwind_jll/Project.toml

```diff
@@ -1,6 +1,6 @@
-name = "LibOSXUnwind_jll"
-uuid = "a83860b7-747b-57cf-bf1f-3e79990d037f"
-version = "0.0.6+1"
+name = "LLVMLibUnwind_jll"
```

Comment:
We have platform dependent stdlibs now?

Reply:
Just like `LibOSXUnwind_jll` this is installed on all platforms but only used on macOS. I will mention that we should be able to use `LLVMLibUnwind_jll`on all platforms in the future but that's beyond the scope of this PR

Reply:
Right it's nothing that is changed in this PR. Who even uses/depends on this stdlib?

Reply:
I believe only the Julia internals depend libunwind. I'm not sure but I suspect the BB infrastructure needs a stdlib in place for the build to work.

Reply:
Okay, seems bad to expose things as official stdlib packages just because BB needs it...

Reply:
@staticfloat would know

Reply:
All binary dependencies must be represented by JLLs for a few reasons:

1) If client code wants to load the library, they should do so by expressing a dependency on a JLL.  This is better than them blindly running `dlopen(libname)` as it won't get confused by `LD_LIBRARY_PATH`, etc....

2) If a user package requires a particular version of a binary dependency, then this will prevent that package from loading on an older version of Julia if that binary dependency is locked to an incompatible versions by Julia itself.  By having explicit JLLs that are stdlibs, we are able to inform the Pkg resolver about the versions of binaries that ship with Julia.

3) For executables that ship with Julia (like `7z`) we can provide the standard tools that users are being trained to use to avoid issues around environment variables, paths, etc...